Sources/SystemModuleKit/Objects/Permission+Objects.swift

```diff
@@ -21,9 +21,9 @@ extension System.Permission {
 
     public struct List: FeatherModuleKit.List {
```

Comment:
WDYT: `FeatherModuleKit.List` -> `ListInterface`? cc @viaszkadi

Reply:
w can rename `FeatherModuleKit.List` to `FeatherModuleKit.ListInterface` to make these interfaces consistent, that would eliminate naming collision as well